### PR TITLE
Update MultiPoly

### PR DESCRIPTION
**Pull request by daviddelaat**

### CODE REVIEW

File: MultiPoly/versions/0.0.2/requires [hunk 1]
```diff
@@ -0,0 +1,3 @@
+julia v0.4
```
Comment (tkelman) [inline MultiPoly/versions/0.0.2/requires]: the v is not necessary and not typically included here

**Comment by tkelman** [comment]
since the previous tag supported julia 0.3 but this tag does not, it would be a good idea to use a new minor package version via `Pkg.tag("MultiPoly", :minor)`

**Comment by daviddelaat** [comment]
`Pkg.publish()` gives an error for me. I'll try again later.

**Comment by daviddelaat** [comment]
Thanks for reviewing. I couldn't get `Pkg.publish()` to work for this package (it does work for a different package), but hope the manual fixes are okay.

**Comment by tkelman** [comment]
Yep. Would Travis CI pass if you enabled it?

**Pull request merged**